src/lib/sw_engine/tvgSwRasterTexmapInternal.h

```diff
@@ -104,6 +104,12 @@
             ab = (int)(255 * (1 - modff(v, &iptr)));
             iru = uu + 1;
             irv = vv + 1;
+
+            //FIXME: If vv goes out of buffer, it can cause potential memory problems.
```

Comment:
i think we can add this condition for the safety. you can remove this comment?

Reply:
I removed comment. Thanks